pallets/pos-switch/src/lib.rs

```diff
@@ -73,7 +130,10 @@ pub mod pallet {
 		/// Switch to PoS
 		#[pallet::call_index(0)]
 		#[pallet::weight((T::BlockWeights::get().max_block, DispatchClass::Operational))]
-		pub fn switch_to_pos(origin: OriginFor<T>) -> DispatchResultWithPostInfo {
+		pub fn switch_to_pos(
+			origin: OriginFor<T>,
+			initial_validators: InitialValidators<T>,
```

Comment:
This will break `scripts/js/src/runtimeUpgrade.ts`

Reply:
Updated